plugins/inputs/varnish/varnish.go

```diff
@@ -22,25 +23,31 @@ type runner func(cmdName string, UseSudo bool, InstanceName string, Timeout inte
 // Varnish is used to store configuration values
 type Varnish struct {
```

Comment:
Add an option of metric_version which defaults to 1. Which `metric_version = 2` adding this new behavior. See the [prometheus input](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/prometheus#configuration) for an example.